profile.php

```diff
@@ -10,8 +10,10 @@
 <body>
 	<?php
 	echo "Welcome\n";
-	echo $_SESSION['name'];
-	echo $_SESSION['class'];
+	echo $_SESSION['name']." ";
+	echo $_SESSION['class']."\n";
+	echo $_SESSION['message'];
+	$_SESSION['message'] = "";
 	?>
 	<a href="lecture.php">hola</a>
```

Comment:
Why hola?

Reply:
Just a random word I like which I put to get to lecture.php to debug it, will change it 😀